lib/migrations/example/index.ts

```diff
@@ -0,0 +1,19 @@
+import { Cli, Commands } from '../src/index';
+
+const cli = new Cli({
+    banner: 'Example database migration tools',
+    name: 'example-cli',
+    knexConfig: {
+        client: 'sqlite3',
+        connection: {
+            filename: `${__dirname}/db.sqlite`,
```

Comment:
We could use an in memory one for speed? ....https://www.sqlite.org/inmemorydb.html

Just a thought.